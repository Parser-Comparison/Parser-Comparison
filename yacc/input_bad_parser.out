> Word: 'abab' --> 9.989738464355469e-05 seconds 

Incorrect, word doesn't belong to language

