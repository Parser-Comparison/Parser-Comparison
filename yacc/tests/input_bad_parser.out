> Word: 'abab' --> 0.00022411346435546875 seconds 

Incorrect, word doesn't belong to language

